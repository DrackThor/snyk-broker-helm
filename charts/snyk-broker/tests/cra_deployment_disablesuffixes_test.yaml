suite: test broker deployment (No suffixes)
templates: 
  - cra_deployment.yaml

tests:
  - it: default values
    values:
      - ./fixtures/default_values_cra_disablesuffixes.yaml
    asserts:
      - matchSnapshot: {}