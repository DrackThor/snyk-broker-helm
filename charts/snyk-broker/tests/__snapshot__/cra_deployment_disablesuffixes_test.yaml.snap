default values:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: RELEASE-NAME-cr
      name: container-registry-agent-cra
      namespace: NAMESPACE
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: RELEASE-NAME-cr
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: RELEASE-NAME-cr
        spec:
          containers:
            - env:
                - name: SNYK_PORT
                  value: "8081"
              image: snyk/container-registry-agent:latest
              imagePullPolicy: Always
              name: container-registry-agent
              ports:
                - containerPort: 8081
                  name: http
              resources:
                limits:
                  cpu: 1
                  memory: 2Gi
                requests:
                  cpu: 1
                  memory: 2Gi
              securityContext:
                allowPrivilegeEscalation: false
                capabilities:
                  drop:
                    - ALL
                readOnlyRootFilesystem: false
                runAsNonRoot: true
                runAsUser: 1000
          securityContext: {}
          serviceAccountName: snyk-broker
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: snyk-broker
        helm.sh/chart: snyk-broker-2.3.0
      name: cra-service
      namespace: NAMESPACE
    spec:
      ports:
        - port: 8081
          targetPort: 8081
      selector:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: RELEASE-NAME-cr
      type: ClusterIP
